_std/code/_macros.dm

```diff
@@ -192,3 +192,5 @@ var/list/detailed_spawn_dbg = list()
 #define num2hex(X, len) num2text(X, len, 16)
 
 #define hex2num(X) text2num(X, 16)
+
+#define reset_anchored(M) do{if(istype(M, /mob/living/carbon/human)){var/mob/living/carbon/human/HumToDeanchor = M;if((HumToDeanchor.shoes && HumToDeanchor.shoes.magnetic) || (HumToDeanchor.mutantrace && HumToDeanchor.mutantrace.anchor_to_floor)){HumToDeanchor.anchored = 1;}else{HumToDeanchor.anchored = 0}}else{M.anchored = 0;}}while(FALSE)
```

Comment:
You can use `\` at the end of a line to make it not be an actual end of the line for the parser. That could help a lot with realiability here.